src/blueapi/core/context.py

```diff
@@ -37,21 +38,44 @@
 LOGGER = logging.getLogger(__name__)
 
 
-@dataclass
 class BlueskyContext:
     """
     Context for building a Bluesky application
     """
 
-    run_engine: RunEngine = field(
-        default_factory=lambda: RunEngine(context_managers=[])
-    )
-    plans: Dict[str, Plan] = field(default_factory=dict)
-    devices: Dict[str, Device] = field(default_factory=dict)
-    plan_functions: Dict[str, PlanGenerator] = field(default_factory=dict)
+    _run_engine: RunEngine
+    _plans: Dict[str, Plan]
+    _devices: Dict[str, Device]
+    _plan_functions: Dict[str, PlanGenerator]
 
     _reference_cache: Dict[Type, Type] = field(default_factory=dict)
```

Comment:
Shouldn't this field be removed as well?

Reply:
Why? It's in use and already private

Reply:
I meant the `= field(default_factory=dict)` part, but it looks like you changed it between me writing and submitting the comment